### PR TITLE
Add inner vlan tag field into APIv2 requests/responses

### PR DESCRIPTION
**Pull request by surabujin**

### CODE REVIEW

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java [hunk 1]
```diff
@@ -64,35 +72,22 @@
     public FlowEndpointPayload(@JsonProperty("switch-id") SwitchId switchId,
                                @JsonProperty("port-id") Integer portId,
                                @JsonProperty("vlan-id") Integer vlanId,
+                               @JsonProperty("inner-vlan-id") Integer innerVlanId,
                                @JsonProperty("detect-connected-devices")
                                            DetectConnectedDevicesPayload detectConnectedDevices) {
         super(switchId, portId);
         setVlanId(vlanId);
+        setInnerVlanId(innerVlanId);
+
```
Comment (niksv) [inline src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java:81]: empty line

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java [hunk 2]
```diff
@@ -149,4 +144,15 @@ public String toString() {
                 .add("detect-connected-devices", detectConnectedDevices)
```
Comment (niksv) [inline src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java:144]: please add inner vlan into toString() and into hashCode()

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java (hunk 1 above)
Comment (surabujin) [reply src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java:81]: em... it looks natural there... removed.

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java (hunk 2 above)
Comment (surabujin) [reply src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/payload/flow/FlowEndpointPayload.java:144]: done

File: src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/model/FlowDto.java [hunk 3]
```diff
@@ -299,7 +309,7 @@ public FlowDto(String flowId,
                 sourcePort,
                 destinationPort,
                 sourceVlan,
-                destinationVlan,
+                destinationVlan, 0, 0,
```
[the remark this answers is not in the document]
Comment (surabujin) [reply src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/model/FlowDto.java:312]: 0 is invalid VLAN id value, so we can use primitives here (at least it allow us to not add a lot of null checks).

File: src-java/kilda-model/src/main/java/org/openkilda/model/FlowEndpoint.java [hunk 4]
```diff
@@ -16,48 +16,109 @@
...
+                .collect(Collectors.toList());
     }
 
     public static boolean isVlanIdSet(Integer vlanId) {
         return vlanId != null && 0 < vlanId;
     }
+
+    @Override
+    public String toString() {
+        String view = String.format("switchId=\"%s\" port=%d", switchId, portNumber);
```
Comment (nikitamarchenko) [inline src-java/kilda-model/src/main/java/org/openkilda/model/FlowEndpoint.java:105]: You mix 2 concat practice here, implicit use of StringBuilder and string.format. Chose one.

Comment (surabujin) [reply src-java/kilda-model/src/main/java/org/openkilda/model/FlowEndpoint.java]: Done.

File: src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/mapper/RequestedFlowIncrementalMapper.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public abstract class RequestedFlowIncrementalMapper {
+    public static final RequestedFlowIncrementalMapper INSTANCE = Mappers.getMapper(
+            RequestedFlowIncrementalMapper.class);
+
+    @Mapping(source = "switchId", target = "srcSwitch")
+    @Mapping(source = "portNumber", target = "srcPort")
+    @Mapping(source = "outerVlanId", target = "srcVlan")
+    @Mapping(source = "innerVlanId", target = "srcInnerVlan")
+    public abstract void mapSource(@MappingTarget RequestedFlow target, FlowEndpoint endpoint);
```
Comment (sergii-iakovenko) [inline src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/mapper/RequestedFlowIncrementalMapper.java:36]: nit: it's more common to have the target as the last argument. https://mapstruct.org/documentation/stable/reference/html/

**Pull request merged**